predict.py

```diff
@@ -109,18 +109,18 @@ def predict(
             description="Adjusts randomness of outputs, greater than 1 is random and 0 is deterministic, 0.75 is a good starting value.",
             ge=0.01,
             le=5,
-            default=0.75,
+            default=0.7,
         ),
         top_p: float = Input(
             description="When decoding text, samples from the top p percentage of most likely tokens; lower to ignore less likely tokens",
             ge=0.0,
             le=1.0,
-            default=0.9,
+            default=0.95,
         ),
```

Comment:
it'd be better to hide top_k for MLC models with the logic we have for hiding `SYSTEM_PROMPT`. Also not a blocker.